resources/windows/windows.pkr.hcl

```diff
@@ -16,11 +25,23 @@ variable "region" {
 variable "install_user" {
   type    = string
   default = "vagrant"
+  description = <<EOF
+The name of a temporarily created Administrator account used for provisioning
+EOF
 }
 
 variable "install_pass" {
-  type    = string
-  default = "P@ssw0rd"
+  type = string
+  description = <<EOF
+The password of a temporarily created Administrator account used for provisioning - see install_user.
```

Comment:
Again not temporary although does not need to be `known` in future as ssh is used to login based on an ssh-key.